tenets/codelingo/effective-go/package-comment/codelingo.yaml

```diff
@@ -0,0 +1,17 @@
+tenets:
+  - name: package-comment
+    flows:
+      codelingo/review:
+        comment: |
+          Every package should have a package comment, a block comment preceding the package clause.
+          For multi-file packages, the package comment only needs to be present in one file, and any one will do.
+    query: |
+      import codelingo/ast/go
+
+      @review comment
+      go.file(depth = any):
+        exclude:
+          go.comment(depth = any):
```

Comment:
Can you handle multi-file packages? If not, load up a blocked card and add an issue to the Tenet.

Reply:
Give this a go

```
import codelingo/ast/go

@review comment
go.dir(depth = any):
  go.file:
    go.ident:
      name as packageName
    exclude:
      go.comment(depth = any):
        start_offset == 0
  exclude:
    go.file:
      go.ident:
        name == packageName
      go.comment(depth = any):
        start_offset == 0
```